rosidl_generator_cpp/msg/PrimitivesConstants.msg

```diff
@@ -0,0 +1,14 @@
+bool BOOL_CONST=true
+byte BYTE_CONST=50
+char CHAR_CONST=100
+float32 FLOAT32_CONST=1.125
+float64 FLOAT64_CONST=3.14159
```

Comment:
oh sorry I misread your comment, I'll update it the the value I used in `test_communication` for multi_rcl testing. thanks for pointing it out!